src/clib/pio_sdecomps_regex.hpp

```diff
@@ -0,0 +1,197 @@
+#ifndef __PIO_SDECOMPS_REGEX_HPP__
```

Comment:
Is this the only C++ code?

Seems like we could find a C-based regex package just as easily...